src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -124,14 +124,13 @@
 
   <!-- TODO: Remove when all repos use a consistent set of eng/common files: https://github.com/dotnet/source-build/issues/3710. -->
   <Target Name="UpdateEngCommonFiles"
-          Condition="'$(RepositoryName)' != 'arcade' and ('$(UpdateEngCommonFiles)' == 'true' or '$(DotNetBuildVertical)' == 'true')"
+          Condition="'$(UpdateEngCommonFiles)' == 'true' or '$(DotNetBuildVertical)' == 'true'"
```

Comment:
I thought this was going to be updated to use UseBootstrapArcade instead?

Reply:
@mmitche and I chatted offline and we realized that the VMR root's eng/common folder is identical with the one under src/arcade and src/installer. It's also the most up-to-date one which makes it the preferred origin for use in the vertical builds, until we come up with a better design (tracked via https://github.com/dotnet/source-build/issues/3710).

The Copy task skips any files that are identical so this is a no-op for Arcade and Installer.

Reply:
> It's also the most up-to-date one which makes it the preferred origin for use in the vertical builds

You really can't say that for the bootstrap repos given they use the n-1 arcade.  I agree though that the contents will be identical therefore either implementation doesn't change the behavior.  As you said, this will need https://github.com/dotnet/source-build/issues/3710 to solve.